source/docs/appendices/release-notes.md

```diff
@@ -17,6 +17,19 @@ Compatible with:
 - **Magento1**: CE 1.7+, EE 1.12+, [OpenMageLTS](https://www.openmage.org/supported-versions.html) 19.4+
 - **Proximis**: latest version
 
+## 2.13.0 (2022-02-17)
+
+> In this release you can start loging your customers using Facebook and Google!
+> 
+> We added support for HiPay embeded payments
+>
+> Algolia search support improvements for both Magento1 and Magento2!
+>
+> And lets not forget premissions checking to the B2B module.
+
+- [Announcement](https://developers.front-commerce.com/blog/2022/02/17/front-commerce-2.13/)
```

Comment:
Announcement post not here yet. Normal or not?

Reply:
is this somethign we write or we take it from the marketing team and put it as is?

Reply:
updated